.github/ci-prerequisites.sh

```diff
@@ -0,0 +1,7 @@
+# Reclaim disk space, otherwise we only have 13 GB free at the start of a job
+
```

Comment:
add `df -h` to see stats about disk usage before cleanup

Reply:
I've also add an echo message